plugin/forward/forward.go

```diff
@@ -194,6 +196,15 @@ func (f *Forward) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg
 			return 0, nil
 		}
 
+		// Check if we have an alternate Rcode defined, check if we match on the code
+		for _, alternateRcode := range f.alternateRcodes {
+			if alternateRcode == ret.Rcode && f.Next != nil { // In case we do not have a Next handler, just continue normally
+				//				if _, ok := f.Next.(*Forward); ok { // Only continue if the next forwarder is also a Forworder
```

Comment:
This is how I was thinking it would work... only going to next if its another instance of the forward plugin.